Automattic/newspack-popups
# feat: integrate criteria with segments

### All Submissions:






### Changes proposed in this Pull Request:



Changes the Segments schema and store an array of criteria.

Also drafts the migration method, changing the segment structure on the fly.

### How to test the changes in this Pull Request:

1. Checkout and build https://github.com/Automattic/newspack-plugin/pull/2537
2. Create and edit segments. Confirm everything still work as usual
3. Confirm tests are passing

### Other information:
